tests/support/server.tcl

```diff
@@ -31,6 +31,16 @@ proc check_valgrind_errors stderr {
     }
 }
 
+proc clean_binaries config {
```

Comment:
@oranagra I'd consider a different name, these are the persistent files. I was asking myself how come the binaries (executables) end up taking a lot of space until reaching this part...